stacks/signal-processing/nifi.yaml

```diff
@@ -18,6 +20,13 @@ spec:
       keySecret: nifi-sensitive-property-key
       autoGenerate: true
   nodes:
+    podOverrides:
+      metadata:
+        annotations:
+          sidecar.opentelemetry.io/inject: "default/otel-collector-grpc"
+          instrumentation.opentelemetry.io/inject-java: "true"
+          # Todo: We probably want to add the following annotation by default.
+          instrumentation.opentelemetry.io/java-container-names: "nifi"
```

Comment:
Did you mean to check this in?

Reply:
oops, nope. Will remove

Reply:
Done in da1b1fe2762c997807e99ae94eac35f12ae8e5f5